src/librustc/hir/lowering.rs

```diff
@@ -587,14 +587,15 @@ impl<'a> LoweringContext<'a> {
 
         impl<'tcx, 'interner> Visitor<'tcx> for ItemLowerer<'tcx, 'interner> {
             fn visit_mod(&mut self, m: &'tcx Mod, _s: Span, _attrs: &[Attribute], n: NodeId) {
-                self.lctx.modules.insert(n, hir::ModuleItems {
+                let module_id = self.lctx.node_id_to_hir_id[n];
```

Comment:
This should probably call `lower_node_id`?

Reply:
It seems like you unfixed this?

Reply:
You're right, forgot about this one; re-fixed :+1:.